lib/src/main/java/uk/gov/di/ipv/cri/kbv/api/service/ExperianService.java

```diff
@@ -38,9 +38,10 @@ public String getResponseFromKBVExperianAPI(String payload, String uriEndpoint)
         HttpResponse<String> res =
                 this.httpClient.send(httpReq, HttpResponse.BodyHandlers.ofString());
 
-        LOGGER.debug(String.format("KBV Experian API response status code: %s", res.statusCode()));
+        LOGGER.debug(
+                (String.format("KBV Experian API response status code: %s", res.statusCode())));
```

Comment:
Why the extra ()

Reply:
That was typo and now I have actually made that in one line with a condition to check as well to reduce a sonar code smell about having to invoke the logging conditionally